bedrock/firefox/tests/test_firefox_details.py

```diff
@@ -24,27 +27,50 @@
     "de": {"25.0": GOOD_PLATS},
     "fr": {"24.0": GOOD_PLATS},  # prev release
 }
-GOOD_VERSIONS = {
+GOOD_BUILDS__THREE_DIGITS = {
+    "en-US": {
+        "125.0": GOOD_PLATS,  # current release
+        "126.0b2": GOOD_PLATS,
+        "127.0a1": GOOD_PLATS,
+    },
+    "de": {"125.0": GOOD_PLATS},
+    "fr": {"124.0": GOOD_PLATS},  # prev release
+}
+
+GOOD_VERSIONS__TWO_DIGITS = {
     "LATEST_FIREFOX_VERSION": "25.0",
     "LATEST_FIREFOX_DEVEL_VERSION": "26.0b2",
     "FIREFOX_DEVEDITION": "26.0b2",
     "FIREFOX_AURORA": "",
     "FIREFOX_ESR": "24.1.0esr",
 }
-GOOD_VERSIONS_NO_DEV = {
+GOOD_VERSIONS__THREE_DIGITS = {
+    "LATEST_FIREFOX_VERSION": "125.0",
+    "LATEST_FIREFOX_DEVEL_VERSION": "126.0b2",
+    "FIREFOX_DEVEDITION": "126.0b2",
+    "FIREFOX_AURORA": "",
+    "FIREFOX_ESR": "124.1.0esr",
+}
+GOOD_VERSIONS__TWO_DIGITS_NO_DEV = {
     "LATEST_FIREFOX_VERSION": "25.0",
     "LATEST_FIREFOX_DEVEL_VERSION": "26.0b2",
     "FIREFOX_AURORA": "",
     "FIREFOX_ESR": "24.1.0esr",
 }
+GOOD_VERSIONS__THREE_DIGITS_NO_DEV = {
+    "LATEST_FIREFOX_VERSION": "125.0",
+    "LATEST_FIREFOX_DEVEL_VERSION": "126.0b2",
+    "FIREFOX_AURORA": "",
+    "FIREFOX_ESR": "124.1.0esr",
+}
```

Comment:
Aurora is no longer a thing right? Can we just remove those?

Reply:
I think it's  still in product-details data though.